features/ifttt/src/main/java/org/opennms/features/ifttt/config/Trigger.java

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.features.ifttt.config;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+
+public class Trigger {
+    /**
+     * the IFTTT event to send
+     */
+    private String eventName = "";
+    /**
+     * the value1 to be used
+     */
+    private String value1 = "";
+    /**
+     * the value2 to be used
+     */
+    private String value2 = "";
+    /**
+     * the value3 to be used
+     */
+    private String value3 = "";
+    /**
+     * delay after executing the trigger
+     */
+    private int delay;
```

Comment:
There are a lot of comments, but the real useful information if this is in seconds, nanoseconds, milliseconds is missing.